admin/css/picker.css

```diff
@@ -138,12 +144,124 @@
   align-self: flex-end;
 }
 
+.close-icon {
+  background: url("../../images/black-cross.svg");
+  cursor: pointer;
+  height: 8px;
+  width: 8px;
+  background-size: contain;
+}
+
 .picker-sidebar .edit-image {
+  display: flex;
```

Comment:
It seems all of these new properties are not needed for this button, if I remove them nothing changes 🤔 (except for the margin-top and the color changes)